svgscreen/svgscreen.go

```diff
@@ -78,18 +127,61 @@ func Render(w io.Writer, s Screen) error {
 				}
 				l.Chars[i] = c
 			}
+		}
+	}
 
+	// Render the whole background first. Then it will not be selected when copy/pasting from rendered SVG
+	for _, l := range s.Lines {
+		bg := LineToTextElement(s, l, func(c Char) TextSpan {
 			if c.Background == "" {
-				continue
-			}
-			if strings.HasPrefix(c.Background, "#") {
-				backgroundColorsUsed[c.Background] = c.Background
+				return TextSpan{
+					Style:   "",
+					Content: " ",
+				}
 			} else {
-				backgroundColorsUsed[c.Background] = s.BackgroundColors[c.Background]
+				return TextSpan{
+					Style: template.CSS(
+						"fill:" + ResolveColor(c.Background, s.BackgroundColors) + "; text-shadow: -0.01px -0.01px, 0.01px -0.01px, -0.01px 0.01px, 0.01px 0.01px;"),
```

Comment:
The idea is that the shadow will hopefully "fill in" the gap? have some vague memory of struggling with black grid before, could that be why there was some stroke leftovers? hmm

Reply:
yep, but it seems to be complicated. like shadow improves rendering in firefox but can actually make it worse for chrome/safari. maybe make it a cmdline option? also it should be a CSS style instead of being repeated for each background `tspan`.

Reply:
also I thought of actually going back from full block characters to drawn rects :rofl: 
when we can use `ch`/`em` units with rects then they align just as well as the block characters plus we can combine them to avoid the black grid...

Reply:
Ah true :) even back to using xlink? could possibly also consolidate based color and rect bigger rects per line or even more? but maybe it's good to not collect too much into this PR

Reply:
I made a commit to revert the shadow, since it doesn't improve the look in all browsers. we can add it later, connected to a cmdline option, if we don't find a better way getting rid of the black grid.